samples/VanillaJSTestApp/index_authCode.html

```diff
@@ -53,11 +53,15 @@
     // instantiate MSAL
     const myMSALObj = new msal.PublicClientApplication(msalConfig);
     // register callback for redirect usecases
-    myMSALObj.handleRedirectCallback(authRedirectCallBack);
+    // myMSALObj.handleRedirectCallback(authRedirectCallBack);
     // signin and acquire a token silently with POPUP flow. Fall back in case of failure with silent acquisition to popup
     function signIn() {
         try {
-            myMSALObj.loginRedirect(loginRequest);
+            myMSALObj.loginPopup(loginRequest).then(tokenResponse => {
+                console.log("Response: " + JSON.stringify(tokenResponse));
+            }).catch(error => {
+                console.error("Auth Error: " + error);
+            });
         } catch (e) {
```

Comment:
Did you mean to commit these changes to the sample?

Reply:
yes, in order to let people run the popup code and test the changes.